src/util/algolia.js

```diff
@@ -0,0 +1,53 @@
+
+const pt = require('@contentful/rich-text-plain-text-renderer');
+
+const pageQuery = `{
+    articles: allContentfulLearningArticlePage(filter:{node_locale:{eq:"en-US"}}) {
```

Comment:
Ah the ol' `en-US`!  What are plans for localization here?  It seems like we should support full-text search for all our supported locales, but it's understandable that might be out of the scope of this initial functionality.

Reply:
Yes, absolutely! My initial thinking would be to actually generate a separate index on Algolia for each of our locales, which we can do in this `algolia.js` file pretty easily (they actually show an example of generating two search indexes [in the docs](https://www.gatsbyjs.org/docs/adding-search-with-algolia/)). Since we don't have any Spanish content in now, and it's still not officially decided which we are deploying  first (learning center or Spanish localization), I think this may be out of scope for now, but something to definitely ponder over.